Assets/Scripts/Player/Movement/PlayerMovement.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private CharacterController characterController;
+    [SerializeField] private float movementSpeed;
+    
+    private Vector3 _moveDirection;
+    private float _x;
+    private float _z;
```

Comment:
```suggestion
    private float _horizontal;
    private float _vertical;
```

Waarom is dit gerenamed in midden in development?